### PR TITLE
Enable selection of a range in checkboxes lists

### PR DESCRIPTION
**Pull request by cproensa**
Enable selection of a range of checkboxes with shift-click.

Fixes: #25217

### CODE REVIEW

File: js/common.js [hunk 1]
```diff
@@ -175,6 +175,43 @@ $(document).ready( function() {
...
+	$('.checkbox-range-selection').on('click', 'div.checkbox label', function (e) {
+		var jcontainer = $(this).closest('.checkbox-range-selection');
+		var last_clicked = jcontainer.data('checkbox-range-last-clicked');
+		if (!last_clicked) {
+			last_clicked = this;
+		}
+		if (e.shiftKey) {
+			// Because shift-click is triggered in a label/span, some browsers
+			// will activate a text selection. Remove text selection.
+			window.getSelection().removeAllRanges();
```
Comment (atrol) [inline js/common.js]: Confirmed.
It works quite smooth on Firefox und Chrome.
On Safari, IE and Edge I get a flickering, caused by selecting / unselecting the whole text in row

**Comment by cproensa** [comment]
Added an alternate strategy for table arrangements, like manage_config_email_page.

**Comment by dregad** [comment]
@cproensa please fix conflicts

**Comment by cproensa** [comment]
> @cproensa please fix conflicts

rebased

**Pull request merged**